src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -265,10 +265,11 @@ public function setVisibility($path, $visibility)
     /**
      * Prepend to a file.
      *
-     * @param  string  $path
-     * @param  string  $data
-     * @param  string  $separator
+     * @param  string $path
+     * @param  string $data
+     * @param  string $separator
      * @return bool
+     * @throws ContractFileNotFoundException
```

Comment:
This isn't the name of the Exception. The exception is `\Illuminate\Contracts\Filesystem\FileNotFoundException`